javascript/javascript3/week1/weather.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@500&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css
+    "
+    />
+
+    <link rel="stylesheet" href="weather.css" />
+    <link rel="icon" type="image/png" href="images/sun-image.png" />
+
+    <title>Weather app</title>
+  </head>
+  <body>
+    <main>
+      <div class="timer-container">
+        <h1 id="time">12:30 <span id="am-pm">PM</span></h1>
+        <div id="date">
```

Comment:
Always try skipping id, rather use class, unless you must use it, though that is not a common use case.